ingest/clusters.py

```diff
@@ -75,6 +76,32 @@ def __init__(
         self.extra_log_params = {'study_id': self.study_id, 'duration': None}
         self.preprocess()
 
+    # Will evolve to do cross file validation
+    def validate(self):
+        """ Runs all validation checks
+        """
+        return all([self.is_valid_format()])
+
+    def is_valid_format(self):
+        """Validates format by calling all format validation methods"""
+        return all(
+            [self.validate_header_for_coordinate_values(), self.validate_format()]
+        )
+
+    def validate_header_for_coordinate_values(self):
+        """Cluster files must have coordinates 'x' and 'y' in header
+        :return: boolean True if coordinates are in header, otherwise False
+        """
+        lower_cased_headers = [header.lower() for header in self.headers]
+        for coordinate in ('x', 'y'):
+            if coordinate not in lower_cased_headers:
+                msg = (
+                    f"Header must have coordinate values 'x' and 'y' (case insensitive)"
+                )
```

Comment:
No interpolation is being done here, so an f-string seems unnecessary -- a basic string seems simpler and sufficient.  

(I suspect this syntax noise comes from a linter -- and that refining linter settings would fix it.)

```suggestion
                msg = (
                    "Header must have coordinate values 'x' and 'y' (case insensitive)"
                )
```

Reply:
0f2b689